Sources/Foundation/FileManager+Win32.swift

```diff
@@ -198,28 +288,15 @@ extension FileManager {
     }
 
     internal func _contentsOfDir(atPath path: String, _ closure: (String, Int32) throws -> () ) throws {
-        guard path != "" else {
-            throw NSError(domain: NSCocoaErrorDomain, code: CocoaError.fileReadInvalidFileName.rawValue, userInfo: [NSFilePathErrorKey : NSString(path)])
+        guard !path.isEmpty else {
+            throw NSError(domain: NSCocoaErrorDomain,
+                          code: CocoaError.fileReadInvalidFileName.rawValue,
+                          userInfo: [NSFilePathErrorKey:NSString(path)])
```

Comment:
```suggestion
            throw CocoaError.error(.fileReadInvalidFileName, userInfo: [NSFilePathErrorKey: path])
```
Either you _don't_ need the `NSString()` conversion here or you _do_ need it above on line 63, which is it? 😕

Reply:
I think that we can get away without doing the `NSString` bridging.  Let's go with that (assuming that tests don't break).